go-controller/pkg/ovn/egressip_test.go

```diff
@@ -11590,11 +11637,34 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 						Action:   nbdb.LogicalRouterPolicyActionAllow,
 						UUID:     "no-reroute-service-UUID",
 					},
+					&nbdb.LogicalRouterPolicy{
+						Priority: types.EgressIPReroutePriority,
+						Match:    fmt.Sprintf("ip4.src == %s", egressPod1.Status.PodIP),
+						Action:   nbdb.LogicalRouterPolicyActionReroute,
```

Comment:
can we have another node thats also hosting an EIP thats unaffected - i just want to test the scenario where theres more than one next hop, nodes deleted and we need to remove old next hop and add the new one while also preserving the other valid one.